### PR TITLE
cni e2e test

### PR DESCRIPTION
**Pull request by heartlock**
step 1: configure cni in test-e2e.sh
step 2: add cni e2e test case

### CODE REVIEW

**Comment by heartlock** [comment]
wait #73 @resouer @feiskyer

File: test/e2e/cni.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```
Comment (feiskyer) [inline test/e2e/cni.go]: s/2016/2017

File: test/e2e/framework/util.go [hunk 2]
```diff
@@ -107,6 +107,12 @@ func PodFound(podsandboxs []*kubeapi.PodSandbox, podId string) bool {
 	}
 	return false
 }
+func CniWork(podNetworkStatus *kubeapi.PodSandboxNetworkStatus) bool {
+	if podNetworkStatus.Ip != "" {
```
Comment (feiskyer) [inline test/e2e/framework/util.go]: Also checks whether the ip is in subnet `10.10.0.0/16`?

Comment (heartlock) [inline test/e2e/framework/util.go]: updated it @feiskyer

**Comment by feiskyer** [comment]
LGTM. Thanks.

**Pull request merged**